dirsrvtests/tests/suites/password/password_test.py

```diff
@@ -132,6 +146,159 @@ def test_password_modify_non_utf8(topology_st, pbkdf2_sha512_scheme):
 
     log.info('test_password_modify_non_utf8: PASSED')
 
+def test_pwd_scheme_not_upgraded_on_bind(topology_st, crypt_scheme, request):
```

Comment:
FYI: since test_pwd_scheme_not_upgraded_on_bind and test_pwd_scheme_crypt_upgraded_on_bind are very similar, I think you should rather have a single parametrized test 
each parameter values being a tuple (no_upgrade_hash_value, expected_hash_after_bind)